apypie/foreman.py

```diff
@@ -0,0 +1,228 @@
+"""
+Apypie Foreman module
+
+opinionated helpers to use Apypie with Foreman
+"""
+import time
+
+try:
+    from typing import cast, Optional, Set, Tuple
```

Comment:
You use `dict` so can you use `set` and `tuple` instead of importing it from Typing? Or is there another difference?

Reply:
In older Python 3 versions, one can't write `tuple[dict, set[str]]`, but `Tuple[dict, Set[str]]` using the stuff from `typing` works.